src/software/ai/hl/stp/tactic/pass_defender/pass_defender_fsm.cpp

```diff
@@ -47,7 +47,8 @@ void PassDefenderFSM::blockPass(const Update& event)
     // on the field that blocks a passing lane between two enemy robots
     event.common.set_primitive(std::make_unique<MovePrimitive>(
         event.common.robot, position_to_block_from, face_ball_orientation,
-        TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT, TbotsProto::DribblerMode::OFF,
+        TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
+        TbotsProto::ObstacleAvoidanceMode::SAFE, TbotsProto::DribblerMode::OFF,
```

Comment:
why not be aggressive when defending a pass?

Reply:
bump

Reply:
Sorry, missed it the first time. Updated now!